var/spack/repos/builtin/packages/amdfftw/package.py

```diff
@@ -0,0 +1,130 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+from spack import *
+from spack.pkg.builtin.fftw import FftwBase
+
+
+class Amdfftw(FftwBase):
+    """FFTW (AMD Optimized version) is a comprehensive collection of
+    fast C routines for computing the Discrete Fourier Transform (DFT)
+    and various special cases thereof.
+
+    It is an open-source implementation of the Fast Fourier transform
+    algorithm. It can compute transforms of real and complex-values
+    arrays of arbitrary size and dimension.
+    AMD Optimized FFTW is the optimized FFTW implementation targeted
+    for AMD CPUs.
+
+    For \'single\' precision build, please use precision value as float.
+    Example : \'spack install amdfftw precision=float\'
+    """
+
+    _name = 'amdfftw'
+    homepage = "https://developer.amd.com/amd-aocl/fftw/"
+    url = "https://github.com/amd/amd-fftw/archive/2.2.tar.gz"
+    git = "https://github.com/amd/amd-fftw.git"
+
+    maintainers = ['amd-toolchain-support']
+
+    version('2.2', sha256='de9d777236fb290c335860b458131678f75aa0799c641490c644c843f0e246f8')
+
+    variant('shared', default=True, description='Builds a shared version of the library')
+    variant('openmp', default=True, description="Enable OpenMP support")
+    variant('debug', default=False, description='Builds a debug version of the library')
+
+    depends_on('texinfo')
+
+    provides('fftw-api@3', when='@2:')
+
+    conflicts('precision=quad', when='%aocc', msg="AOCC clang doesn\'t support quad precision")
+    conflicts('+debug', when='%aocc', msg="AOCC clang doesn\'t support debug")
+    conflicts('%gcc@:7.2', when="@2.2:", msg="Recommended GCC version above 7.2 for AMDFFTW")
```

Comment:
Is this a recommendation or a requirement? `conflicts` is a hard requirement. If it's only a recommendation, I would remove this line.

Reply:
Rewording done from 'Recommended' to 'Required'